### PR TITLE
Who is using wechaty subsection of the introduction section

### PR DESCRIPTION
**Pull request by nibble0101**
# Improve the Introduction and Explanation sections project

| Section | Sub-section | Technical Writer |
| --- | --- | --- |
| Introduction | Who is using wechaty |  Chris Estepa, Joseph Mawa |

This  is **Who is using wechaty** sub-section of the **Introduction** section

image
image
image

### CODE REVIEW

File: docusaurus/docs/who-is-using-wechaty.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+- [L'Oréal](https://www.loreal.com/en/) - the world leader in beauty and makeup
+- [Xueersi](https://www.xueersi.com/) - a Beijing-based educational training site focused on primary and middle school students
+- [58Tong Cheng](https://by.58.com/) - is China's leading life service platform which provides different types of services such as recruitment, domestic, and rental services
+- [Plug And Play](https://www.plugandplaytechcenter.com/) - is the world's largest early stage investor, accelerator, and corporate innovation platform
+- [Y Combinator](https://www.ycombinator.com/) - is a venture capitalist that provides seed funding for startups
+
+If you are interested in using Wechaty, you are welcome to join our [community](#link) of users and developers. We are an inclusive and welcoming community.
+If you are still not sure whether Wechaty can solve your problem or challenge, take a look at [what you can do with Wechaty](#link) section. Wechaty is evolving very fast. New Wechaty [puppets](#link-to-puppets) and [plugins](#link-to-plugins) are being added by our dedicated community of developers. Feel free to ask. We shall be happy to help you.
```
Comment (iamrajiv) [inline docusaurus/docs/who-is-using-wechaty.md]:
```suggestion
If you are still unsure whether Wechaty can solve your problem or challenge, take a look at [what you can do with Wechaty](#link) section. Wechaty is evolving very fast. Our dedicated community of developers is adding new Wechaty [puppets](#link-to-puppets) and [plugins](#link-to-plugins). Feel free to ask. We shall be happy to help you.
```

File: docusaurus/docs/who-is-using-wechaty.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+- [Aliyun](https://eu.alibabacloud.com/en) - is China's leading cloud provider, also known as Alibaba Cloud
+- [Meituan](https://www.meituan.com/) - is a group-discount website which sells vouchers from merchants for deals and generates most of its revenue from mobile application services.
+- [L'Oréal](https://www.loreal.com/en/) - the world leader in beauty and makeup
+- [Xueersi](https://www.xueersi.com/) - a Beijing-based educational training site focused on primary and middle school students
+- [58Tong Cheng](https://by.58.com/) - is China's leading life service platform which provides different types of services such as recruitment, domestic, and rental services
+- [Plug And Play](https://www.plugandplaytechcenter.com/) - is the world's largest early stage investor, accelerator, and corporate innovation platform
+- [Y Combinator](https://www.ycombinator.com/) - is a venture capitalist that provides seed funding for startups
+
+If you are interested in using Wechaty, you are welcome to join our [community](#link) of users and developers. We are an inclusive and welcoming community.
```
Comment (iamrajiv) [inline docusaurus/docs/who-is-using-wechaty.md]:
```suggestion
If you are interested in using Wechaty, you are welcome to join our [community](#link) of users and developers. We are an inclusive and welcoming community.

```

File: docusaurus/docs/who-is-using-wechaty.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+## Companies using Wechaty
+
+Some of the companies that are currently using Wechaty include:
+
+- [Wechat](https://www.wechat.com/en/) - is China's top free messaging and calling app that serves as a mobile internet
+- [BAIDU](http://www.baidu.com/) - a Chinese multinational technology company specializing in Internet-related services, products, and artificial intelligence
+- [VIP.COM](https://www.vip.com/) and [JD.COM](https://global.jd.com/) - famous online shops in China selling lifestyle goods
+- [Tencent](https://www.tencent.com/en-us/) - a Chinese multinational technology conglomerate holding company
+- [Aliyun](https://eu.alibabacloud.com/en) - is China's leading cloud provider, also known as Alibaba Cloud
+- [Meituan](https://www.meituan.com/) - is a group-discount website which sells vouchers from merchants for deals and generates most of its revenue from mobile application services.
```
Comment (iamrajiv) [inline docusaurus/docs/who-is-using-wechaty.md]:
```suggestion
- [Meituan](https://www.meituan.com/) - is a group-discount website that sells vouchers from merchants for deals and generates most of its revenue from mobile application services.
```

This fixes the pronoun problem.

File: docusaurus/docs/who-is-using-wechaty.md [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+- [L'Oréal](https://www.loreal.com/en/) - the world leader in beauty and makeup
+- [Xueersi](https://www.xueersi.com/) - a Beijing-based educational training site focused on primary and middle school students
+- [58Tong Cheng](https://by.58.com/) - is China's leading life service platform which provides different types of services such as recruitment, domestic, and rental services
+- [Plug And Play](https://www.plugandplaytechcenter.com/) - is the world's largest early stage investor, accelerator, and corporate innovation platform
```
Comment (iamrajiv) [inline docusaurus/docs/who-is-using-wechaty.md]:
```suggestion
- [Plug And Play](https://www.plugandplaytechcenter.com/) - is the world's largest early-stage investor, accelerator, and corporate innovation platform
```

**Review by iamrajiv — CHANGES_REQUESTED** [review]

**Review by iamrajiv — APPROVED** [review]

**Comment by proudofsimin** [comment]
Hi @nibble0101,
I guess this part also includes a bit of marketing purpose. Imagine yourself as a user who is a foreigner and has never used Wechaty before. When you are considering if you would like to import this software to your company, seeing that most of the users are Chinese companies, how do you feel? Although your introduction is correct, but it makes Wechaty seems a bit local.  As far as I know, those companies have many users outside China. Do you think if it's better that we focus them on the international part than where they are based? For example like L'Oréal is from France but you would describe it as "world's leading FMCG". 
This is my opinions. How do you think? And @huan I'm not sure if this is how you like the customers to be described or if they have specific official introduction.

**Comment by chris-4444** [comment]
@huan Can you please confirm what L'Oréal branch is using Wechaty just to be sure? Is it L'Oréal China?
@proudofsimin Would it be okay if we put L'Oréal China there once it was confirmed? Or, would you like us to remove the company descriptions? All are Chinese companies except for Plug and Play, which is an international company.

**Comment by nibble0101** [comment]
> Hi @nibble0101,
> I guess this part also includes a bit of marketing purpose. Imagine yourself as a user who is a foreigner and has never used Wechaty before. When you are considering if you would like to import this software to your company, seeing that most of the users are Chinese companies, how do you feel? Although your introduction is correct, but it makes Wechaty seems a bit local. As far as I know, those companies have many users outside China. Do you think if it's better that we focus them on the international part than where they are based? For example like L'Oréal is from France but you would describe it as "world's leading FMCG".
> This is my opinions. How do you think? And @huan I'm not sure if this is how you like the customers to be described or if they have specific official introduction.

Thanks @proudofsimin  for the invaluable feedback. Definitely if I were a businessman seeking to start using Wechaty in my business, I would have a lot more confidence in Wechaty if its users are companies that have a global footprint. We can add more information about  each company's  reach (customer base) outside China to give a lot more confidence to potential users, especially those from outside China.  Perhaps @huan  will have to furnish us with a list of other companies outside China that have used or are using Wechaty if they exist so that we have a mix of both China based companies and those outside China.

**Comment by proudofsimin** [comment]
> > Hi @nibble0101,
> > I guess this part also includes a bit of marketing purpose. Imagine yourself as a user who is a foreigner and has never used Wechaty before. When you are considering if you would like to import this software to your company, seeing that most of the users are Chinese companies, how do you feel? Although your introduction is correct, but it makes Wechaty seems a bit local. As far as I know, those companies have many users outside China. Do you think if it's better that we focus them on the international part than where they are based? For example like L'Oréal is from France but you would describe it as "world's leading FMCG".
> > This is my opinions. How do you think? And @huan I'm not sure if this is how you like the customers to be described or if they have specific official introduction.
> 
> Thanks @proudofsimin for the invaluable feedback. Definitely if I were a businessman seeking to start using Wechaty in my business, I would have a lot more confidence in Wechaty if its users are companies that have a global footprint. We can add more information about each company's reach (customer base) outside China to give a lot more confidence to potential users, especially those from outside China. Perhaps @huan will have to furnish us with a list of other companies outside China that have used or are using Wechaty if they exist so that we have a mix of both China based companies and those outside China.

I know those companies are very large and I actually meant that to focus those companies on the international part. For example like Tencent, we could describe it as "a ~Chinese~ multinational technology...". How do you think?

**Comment by nibble0101** [comment]
> > > Hi @nibble0101,
> > > I guess this part also includes a bit of marketing purpose. Imagine yourself as a user who is a foreigner and has never used Wechaty before. When you are considering if you would like to import this software to your company, seeing that most of the users are Chinese companies, how do you feel? Although your introduction is correct, but it makes Wechaty seems a bit local. As far as I know, those companies have many users outside China. Do you think if it's better that we focus them on the international part than where they are based? For example like L'Oréal is from France but you would describe it as "world's leading FMCG".
> > > This is my opinions. How do you think? And @huan I'm not sure if this is how you like the customers to be described or if they have specific official introduction.
> > 
> > 
> > Thanks @proudofsimin for the invaluable feedback. Definitely if I were a businessman seeking to start using Wechaty in my business, I would have a lot more confidence in Wechaty if its users are companies that have a global footprint. We can add more information about each company's reach (customer base) outside China to give a lot more confidence to potential users, especially those from outside China. Perhaps @huan will have to furnish us with a list of other companies outside China that have used or are using Wechaty if they exist so that we have a mix of both China based companies and those outside China.
> 
> I know those companies are very large and I actually meant that to focus those companies on the international part. For example like Tencent, we could describe it as "a ~Chinese~ multinational technology...". How do you think?

That is fine with me. We can do the same for other companies that have footprint outside China as well. What do you think @chris-4444?

**Comment by proudofsimin** [comment]
> @huan Can you please confirm what L'Oréal branch is using Wechaty just to be sure? Is it L'Oréal China?
> @proudofsimin Would it be okay if we put L'Oréal China there once it was confirmed? Or, would you like us to remove the company descriptions? All are Chinese companies except for Plug and Play, which is an international company.

@chris-4444 Wechat, baidu, and those companies are very large in China and have branches all over the world, so maybe we could focus on the international part of them. What I mentioned about L'Oréal means that although L'Oréal is based in France, you would describe it as a world's leading FMCG instead of a French leading FMCG. Do you feel it is better? This is just my point of view. If you feel the original version is better, you could keep it.

**Review by chris-4444 — CHANGES_REQUESTED** [review]
@nibble0101 yep sure. 👍🏽 @proudofsimin yeah you're right.  updated it and now, there's a global feel to it compared before. we'll update again if needed. :)

**Comment by proudofsimin** [comment]
> @nibble0101 yep sure. 👍🏽 @proudofsimin yeah you're right. updated it and now, there's a global feel to it compared before. we'll update again if needed. :)

Hi @chris-4444,
Thanks I saw your commit, but did you check the rest Chinese companies (other than Tencent)? Do they have branches/ market share abroad? 
BTW, I just found out that Wechat is actually owned by Tencent, like WhatsApp to Facebook. You might need to mention it.
Please let me know your opinions.
Thank you :)

**Comment by chris-4444** [comment]
@proudofsimin Updated Tencent description. I noticed that Baidu has a branch in US but didn't mention it. Re the market share, I don't know how to compute in case we need to put it. For the rest of the companies, it is only Tencent and Wechat that are related. :) Should I mention the other locations of the companies then? Tia for the advice.

**Review by Rohitesh-Kumar-Jain — APPROVED** [review]
Looks good to me : )

> Should I mention the other locations of the companies then? Tia for the advice.
@proudofsimin 

Feel free to add ready-to-merge label as soon as Simin approves these changes

**Comment by lijiarui** [comment]
How about we put this to the `Introduction--> community` part instead of putting it just under the `introduction` part?

**Review by lijiarui — COMMENTED** [review]
How about we put this to the `Introduction--> community` part instead of putting it just under the `introduction` part?

**Review by proudofsimin — APPROVED** [review]
Approved for the document content. Regarding the location, I'm fine with either of them. Thank you.

**Comment by nibble0101** [comment]
> How about we put this to the `Introduction--> community` part instead of putting it just under the `introduction` part?

Hi @lijiarui Thanks for the review and sorry for the late reply. Actually it was intentional that we placed that section under Introduction. Just like we mentioned in our proposal, one of our goals is to avoid having documents  nested two or more levels deep especially in the sections we are working on. One of the reasons is to help anyone who is new to easily access the information they are looking for. One example is when  we were writing the GSoD proposals, I had to always use the search bar to locate the Wechaty GSOD proposal because the information is hidden in Introduction -> Community -> Google season of docs. Such nesting makes it difficult to locate specific information in the documentation.

We are also proposing the removal of the community section from the documentation. We don't see any valid reason why it should be in the docs.  The documentation in our opinion looks huge because of all the information we believe should not be there in the first place e.g. community, showcase etc. Even when you check most  of the popular open source projects I have listed below, their community section is not part of the documentation. It is a separate menu item on the website. Doing that will keep the doc clean with only the necessary information so that locating what you need becomes easy. You can check the following docs to see what I am talking about.
- [react native doc](https://reactnative.dev/)
- [electron doc](https://www.electronjs.org/docs)
- [react doc](https://reactjs.org/)
- [docusaurus doc](https://docusaurus.io/)

Since the PRs here are not being merged, we have deployed a clone of  wechaty.js.org containing the Introduction and Explanation section. You can check it out [here](https://wechaty-js-org.netlify.app/docs/). That is how we would love those two sections to be; short and precise with only the necessary information. However if you still want the community section to be part of the docs, we shall add it and include who is using Wechaty section in it.

**Comment by lijiarui** [comment]
> @huan Can you please confirm what L'Oréal branch is using Wechaty just to be sure? Is it L'Oréal China?
> @proudofsimin Would it be okay if we put L'Oréal China there once it was confirmed? Or, would you like us to remove the company descriptions? All are Chinese companies except for Plug and Play, which is an international company.

Hi @chris-4444 , L'Oréal is [Juzi.bot](https://pre-angel.com/portfolios/juzibot/)  customer, my company [Juzi.bot](https://pre-angel.com/portfolios/juzibot/) build on top of Wechaty, we provide SaaS solutions to large enterprises. It is L'Oréal, so I think it is suitable putting L'Oréal on wechaty list.

**Comment by chris-4444** [comment]
LGTM.

**Review by chris-4444 — APPROVED** [review]
LGTM.

**Comment by Rohitesh-Kumar-Jain** [comment]
@chris-4444 and @nibble0101 would you like to push any more commits to this PR?

@nibble0101 approve this PR when you feel it's ready to be merged

**Comment by nibble0101** [comment]
> @chris-4444 and @nibble0101 would you like to push any more commits to this PR?
> 
> @nibble0101 approve this PR when you feel it's ready to be merged

I authored all the PRs. I don't think I will be able to approve any of them. Since @chris-4444  has approved it, then we are good. LGTM

**Comment by Rohitesh-Kumar-Jain** [comment]
> > @chris-4444 and @nibble0101 would you like to push any more commits to this PR?
> > @nibble0101 approve this PR when you feel it's ready to be merged
> 
> I authored all the PRs. I don't think I will be able to approve any of them. Since @chris-4444 has approved it, then we are good. LGTM

thanks for letting me know this : )

File: docusaurus/docs/who-is-using-wechaty.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+- [Tencent](https://www.tencent.com/en-us/) - a multinational technology conglomerate holding company, which also owns the widely used Wechat app
+- [Aliyun](https://eu.alibabacloud.com/en) - is China's leading cloud provider, also known as Alibaba Cloud
+- [Meituan](https://www.meituan.com/) - is a group-discount website which sells vouchers from merchants for deals and generates most of its revenue from mobile application services.
+- [L'Oréal](https://www.loreal.com/en/) - is a leading personal care and cosmetics company
+- [Xueersi](https://www.xueersi.com/) - a Beijing-based educational training site focused on primary and middle school students
+- [58Tong Cheng](https://by.58.com/) - is China's leading life service platform which provides different types of services such as recruitment, domestic, and rental services
+- [Plug And Play](https://www.plugandplaytechcenter.com/) - is the world's largest early-stage investor, accelerator, and corporate innovation platform
+- [Y Combinator](https://www.ycombinator.com/) - is a venture capitalist that provides seed funding for startups
+
```
Comment (lijiarui) [inline docusaurus/docs/who-is-using-wechaty.md]: How do you think of we add this part here： [Voice of Developers](https://github.com/wechaty/wechaty#yum-voice-of-developers)

File: docusaurus/docs/who-is-using-wechaty.md [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+- Conversational AI developers
+- Machine learning researchers
+- Freelancers who are building chatbots for their clients
+- Companies that are building commercial services on top of conversational automation
+
+## Companies using Wechaty
+
+Some of the companies that are currently using Wechaty include:
+
+- [Wechat](https://www.wechat.com/en/) - is China's top free messaging and calling app that serves as a mobile internet
```
Comment (lijiarui) [inline docusaurus/docs/who-is-using-wechaty.md]: Hi @nibble0101 How do you think we change the company introduction to the company logo?  Maybe people don't care what company introduction here.

File: docusaurus/docs/who-is-using-wechaty.md [hunk 7]
```diff
@@ -0,0 +1,33 @@
...
+
+## Users of Wechaty
+
+We have a diverse community of Wechaty contributors and users. The Wechaty users in our community mainly comprise of:
+
+- Conversational AI developers
+- Machine learning researchers
+- Freelancers who are building chatbots for their clients
+- Companies that are building commercial services on top of conversational automation
+
```
Comment (lijiarui) [inline docusaurus/docs/who-is-using-wechaty.md:16]: We have two images introducing wechaty contributors and developers at [wechaty introduction slide](https://docs.google.com/presentation/d/1eRNrKnCpdnsmplTwtZzmtGZgrPoNCmOnitmHKVc6iVU/edit#slide=id.g476e76b184_1_58), do you think it is suitable adding them here?

And I think maybe we can add the following users:
- Google，Tencent , ByteDance engineers and entrepreneurs

image

image

File: docusaurus/docs/who-is-using-wechaty.md [hunk 8]
```diff
@@ -0,0 +1,33 @@
...
+- [Aliyun](https://eu.alibabacloud.com/en) - is China's leading cloud provider, also known as Alibaba Cloud
+- [Meituan](https://www.meituan.com/) - is a group-discount website which sells vouchers from merchants for deals and generates most of its revenue from mobile application services.
+- [L'Oréal](https://www.loreal.com/en/) - is a leading personal care and cosmetics company
+- [Xueersi](https://www.xueersi.com/) - a Beijing-based educational training site focused on primary and middle school students
+- [58Tong Cheng](https://by.58.com/) - is China's leading life service platform which provides different types of services such as recruitment, domestic, and rental services
+- [Plug And Play](https://www.plugandplaytechcenter.com/) - is the world's largest early-stage investor, accelerator, and corporate innovation platform
+- [Y Combinator](https://www.ycombinator.com/) - is a venture capitalist that provides seed funding for startups
+
+If you are interested in using Wechaty, you are welcome to join our [community](#link) of users and developers. We are an inclusive and welcoming community.
```
Comment (lijiarui) [inline docusaurus/docs/who-is-using-wechaty.md]: I think we should change [community](#link) to `[community](https://gitter.im/wechaty/wechaty)`

File: docusaurus/docs/who-is-using-wechaty.md [hunk 9]
```diff
@@ -0,0 +1,33 @@
...
+If you are still not sure whether Wechaty can solve your problem or challenge, take a look at [what you can do with Wechaty](#link) section. Wechaty is evolving very fast. New Wechaty [puppets](#link-to-puppets) and [plugins](#link-to-plugins) are being added by our dedicated community of developers. Feel free to ask. We shall be happy to help you.
```
Comment (lijiarui) [inline docusaurus/docs/who-is-using-wechaty.md]: It seems we lack several links here, how about file a issue to track this, and added in the future?

**Review by lijiarui — CHANGES_REQUESTED** [review]
I have some suggestion about this part, how do you think of this?

**Review by chris-4444 — CHANGES_REQUESTED** [review]
@nibble0101 Updated **If you are interested in using Wechaty** links, last paragraph. Is this okay?

**Comment by nibble0101** [comment]
Hi @lijiarui Thanks for the suggestions. I will look into what you have suggested in the coming days. I think they will make excellent additions to that sub-section. About the links, we shall add those links which point to other sections in the docs after most of the PRs have been merged because they are being worked on by other teams to avoid failing CI. We are tracking them.

@chris-4444  We still have to add some of images suggested by @lijiarui.

**Comment by lijiarui** [comment]
Thanks for your work! Great to know that you are tracking for the links!

File: docusaurus/docs/who-is-using-wechaty.md (hunk 9 above)
Comment (nibble0101) [reply docusaurus/docs/who-is-using-wechaty.md]: Issue #1091 is now tracking the missing the links

**Review by chris-4444 — CHANGES_REQUESTED** [review]
@nibble0101 Added the company logos.  @lijiarui  Would it be okay if we retain the company descriptions because there might be some users who would like to know what these companies do? Even if most clients are from Asia, Wechaty's audience is all over the world so they might want to know these? 😉

File: docusaurus/docs/who-is-using-wechaty.md (hunk 5 above)
Comment (nibble0101) [reply docusaurus/docs/who-is-using-wechaty.md]: Added.

File: docusaurus/docs/who-is-using-wechaty.md (hunk 7 above)
Comment (nibble0101) [reply docusaurus/docs/who-is-using-wechaty.md:16]: Added

File: docusaurus/docs/who-is-using-wechaty.md (hunk 9 above)
Comment (nibble0101) [reply docusaurus/docs/who-is-using-wechaty.md]: Added. Only one missing link awaiting new sections which are yet to be merged.

File: docusaurus/docs/who-is-using-wechaty.md (hunk 8 above)
Comment (nibble0101) [reply docusaurus/docs/who-is-using-wechaty.md]: Added

**Review by chris-4444 — APPROVED** [review]
LGTM.

**Comment by Rohitesh-Kumar-Jain** [comment]
This PR has conflicts but version conflicts can be resolved after @lijiarui again review this

File: docusaurus/docs/who-is-using-wechaty.md [hunk 10]
```diff
@@ -0,0 +1,61 @@
+---
+title: Who is using Wechaty
+---
+
+![Wechaty: Conversational RPA SDK for Chatbot Makers](/img/wechaty-logo.svg)
```
Comment (lijiarui) [inline docusaurus/docs/who-is-using-wechaty.md]: Maybe we should change a more related image here or remove it. Wechaty logo is too general for this part.

File: docusaurus/docs/who-is-using-wechaty.md [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+>
+> "If you know js ... try Wechaty, it's easy to use."  
+> &mdash; <cite>@Urinx Uri Lee, Author of [WeixinBot(Python)](https://github.com/Urinx/WeixinBot)</cite>
+>
+> "Wechaty is a good project, I hope it can continue! Therefore, I became a contributors in open collective."  
+> &mdash; <cite>[@Simple](https://github.com/mrwhh)</cite>
+
+See more at [Wiki:Voice Of Developer](https://github.com/Wechaty/wechaty/wiki/Voice%20Of%20Developer)
+
+If you are interested in using Wechaty, you are welcome to join our [community](https://gitter.im/wechaty/wechaty) of users and [developers](https://github.com/wechaty/wechaty#yum-voice-of-developers). We are an inclusive and welcoming community.
```
Comment (lijiarui) [inline docusaurus/docs/who-is-using-wechaty.md]: Please change developer link to https://wechaty.js.org/contributors/

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: docusaurus/docs/who-is-using-wechaty.md (hunk 10 above)
Comment (nibble0101) [reply docusaurus/docs/who-is-using-wechaty.md]: Removed Wechaty logo.

File: docusaurus/docs/who-is-using-wechaty.md (hunk 11 above)
Comment (nibble0101) [reply docusaurus/docs/who-is-using-wechaty.md]: Change link URL.

File: docusaurus/docs/who-is-using-wechaty.md [hunk 12]
```diff
@@ -0,0 +1,59 @@
...
+- **[Aliyun](https://eu.alibabacloud.com/en)** - is China's leading cloud provider, also known as Alibaba Cloud
+- **[Meituan](https://www.meituan.com/)** - is a group-discount website which sells vouchers from merchants for deals and generates most of its revenue from mobile application services
+- **[L'Oréal](https://www.loreal.com/en/)** - is a leading personal care and cosmetics company
+- **[Xueersi](https://www.xueersi.com/)** - a Beijing-based educational training site focused on primary and middle school students
+- **[58 Tongcheng](https://by.58.com/)** - is China's leading life service platform which provides different types of services such as recruitment, domestic, and rental services
+- **[Plug And Play](https://www.plugandplaytechcenter.com/)** - is the world's largest early-stage investor, accelerator, and corporate innovation platform
+- **[Y Combinator](https://www.ycombinator.com/)** - is a venture capitalist that provides seed funding for startups
+- **Google，Tencent, ByteDance engineers and entrepreneurs**
+  ![Wechaty users](../static/img/docs/wechaty-users.webp)
```
Comment (lijiarui) [inline docusaurus/docs/who-is-using-wechaty.md]: Here is the Wechaty user instead of companies, so I suggest putting them on the `Users of Wechaty` part.

**Review by lijiarui — CHANGES_REQUESTED** [review]

File: docusaurus/docs/who-is-using-wechaty.md (hunk 12 above)
Comment (nibble0101) [reply docusaurus/docs/who-is-using-wechaty.md]: Added to Users of Wechaty section.

**Review by lijiarui — APPROVED** [review]
LGTM

**Comment by Rohitesh-Kumar-Jain** [comment]
@nibble0101 pls  resolve the conflict, and then ping for merge

**Comment by nibble0101** [comment]
> @nibble0101 pls resolve the conflict, and then ping for merge

@Rohitesh-Kumar-Jain  I think we are good.

**Pull request merged**